doc/plugin/vifm-plugin.txt

```diff
@@ -70,6 +70,7 @@ executable.
 
                                                *g:vifm_exec_args*
 A string variable that specifies arguments for vifm.  Empty by default.
+E.g. to start with a single window, instead of two: `let vifm_exec_args='-c only'`
```

Comment:
Could you reformat this as `let loaded_vifm=1` example above?  This line is too long.